packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/error.ts

```diff
@@ -45,6 +45,7 @@ export function processRegistryErrorList(
       extra: {
         outboundRequest,
         registryErrorList,
+        outboundRequestId: outboundRequest.id,
```

Comment:
nit: this is our `requestId`, right? It would be good to use the same name throughout the platform for consistency and easy of understanding.